action.yml

```
@@ -16,6 +16,9 @@ inputs:
   setup-python:
     description: 'If true, Python will automatically be installed on the runner.'
     default: true
+  upgrade:
+    description: 'If true, Linode CLI and its dependencies will be automatically upgraded. (--upgrade)'
+    default: false
```

```suggestion
    default: true
```
I think default to true may be better because we always wish the user to use the latest cli?